packages/server/graphql/private/queries/suProOrgInfo.ts

```diff
@@ -0,0 +1,32 @@
+import getRethink from '../../../database/rethinkDriver'
+import {RValue} from '../../../database/stricterR'
+import {requireSU} from '../../../utils/authorization'
+import {QueryResolvers} from '../resolverTypes'
+
+const suProOrgInfo: QueryResolvers['suProOrgInfo'] = async (
+  _source,
+  {includeInactive},
+  {authToken}
+) => {
+  const r = await getRethink()
+
+  // AUTH
+  requireSU(authToken)
```

Comment:
I didn't change any part of this resolve fn.
this auth check is removed in the next PR #6330 because auth is now handled in `permissions.ts`.